helm/net-exporter/values.yaml

```diff
@@ -12,6 +12,14 @@ dns:
   namespace: kube-system
   service: coredns
 
+dnscache:
+  port: 53
```

Comment:
Maybe it would make more sense to add an array of ports and protocols to open instead and add comments what they are good for?